molSimplify/__main__.py

```diff
@@ -99,8 +105,8 @@ def deprecated_wrapper(func):
 
 try:
     import PyQt5
-    from PyQt5.QtGui import *
-    from molSimplify.Classes.mGUI import *
+    from PyQt5.QtGui import QApplication
```

Comment:
```suggestion
    from PyQt5.QtWidgets import QApplication
```

Reply:
Sorry for not testing the GUI and thanks for fixing!